src/components/table/table.js

```diff
@@ -1338,7 +1338,7 @@ export default {
         )
         // Row Details slot
         if (rowShowDetails) {
-          const tdAttrs = { colspan: String(fields.length) }
+          const tdAttrs = { colspan: String(fields.length), class: 'col-12' }
```

Comment:
This will probably add unexpected padding on the spanned column (`td`) which people may not be expecting.

It should be conditional, or we should add an option to apply additional classes to the details `td`

Reply:
Also note that not everyone uses 12 columns in their custom bootstrap set up (I.e. 10 columns).